src/Repository/WorklogRepository.php

```diff
@@ -48,8 +48,12 @@ public function findByFilterData(Project $project, InvoiceEntry $invoiceEntry, I
 
         $qb->where('worklog.project = :project')->setParameter('project', $project);
 
-        if (isset($filterData->isBilled) && $filterData->isBilled) {
-            $qb->andWhere('worklog.isBilled = 1');
+        if (isset($filterData->isBilled)) {
+            if ($filterData->isBilled) {
+                $qb->andWhere('worklog.isBilled = true');
+            } else {
+                $qb->andWhere('worklog.isBilled = 0 OR worklog.isBilled is NULL');
+            }
```

Comment:
Using `true` and `NULL` seems case-sensitively inconsistent.

Can `worklog.isBilled = 0 OR worklog.isBilled is NULL` be shortened to `worklog.isBilled != TRUE` in DQL? This would prevent having to assume something about when a value is not true.

I (and other pedantics) would write this as

```php
$qb->andWhere(
  $filterData->isBilled
  ? 'worklog.isBilled = true'
  : 'worklog.isBilled = 0 OR worklog.isBilled is NULL' // or 'worklog.isBilled !=true'
);
```